enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/LdapRealm.java

```diff
@@ -224,6 +337,12 @@ private void configureRealm( Config config )
                 parseGroupToRoleMapping( config.get( SecuritySettings.ldap_authorization_group_to_role_mapping ) );
     }
 
+    private String parseLdapServerUrl( String rawLdapServer )
+    {
+        return (rawLdapServer == null) ? null :
```

Comment:
No, not t usually if the setting has a default value. In this case it was just fixing unit tests that used mocked config. It could also have been handled in the tests.